### ISSUE TITLE
Node logs to /dev/null

### ISSUE THREAD

**Issue by Unkn8wn69** [hundehausen/monero-suite#6]
Plesse add a option to change --log-file to /dev/null to enhance node security.

--log-file=/dev/null

**Comment by hundehausen** [comment]
Yes, I like the idea.
Have you tested this config? Does the logs still appear in docker logs?

**Comment by Unkn8wn69** [comment]
No, never tested this with docker, but the log config in docker compos should take care of that ig

**Comment by hundehausen** [comment]
Fixed: Monero node has a "Disable Logging" option that sets `--log-file=/dev/null`, `--max-log-file-size=0`, and Docker `logging: driver: none`.
